apps/shooting-clubs/acceptance/features/location-address.js

```diff
@@ -0,0 +1,220 @@
+'use strict';
+
+const steps = require('../../index');
+
+Feature('Shooting club - location address');
+
+Before((
+  I,
+  locationAddressPage
+) => {
+  I.visitPage(locationAddressPage, steps)
+});
+
+// Location postcode step
```

Comment:
it's just there are so many tests in this file so it's hard to find what is what and isolate things.  I thought the comment will be helpful.

No worries, I can delete if necessary